jetty-ee11/jetty-ee11-servlet/src/main/java/org/eclipse/jetty/ee11/servlet/Dispatcher.java

```diff
@@ -319,15 +320,15 @@ public String getRequestURI()
         @Override
         public StringBuffer getRequestURL()
         {
-            return _uri == null ? super.getRequestURL() : new StringBuffer(HttpURI.build(_uri).query(null).scheme(super.getScheme()).host(super.getServerName()).port(super.getServerPort()).asString());
+            return _uri == null ? super.getRequestURL() :  new StringBuffer(HttpURI.build(_uri).query(null).scheme(super.getScheme()).host(super.getServerName()).port(super.getServerPort()).asString());
```

Comment:
I'm surprised checkstyle doesn't complain about this extra unneeded space.

Reply:
Fake news, I can't see it in the file.